app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class UserMailer < ApplicationMailer
+  default from: 'support@ckblogs.com'
+
+  def welcome_email
+    @user = params[:user]
+    @url  = 'http://localhost:3000/users/sign_in'
```

Comment:
remove that localhost:3000, instead use a variable for development/staging/production URL